packages/legacy-scripting-runner/middleware-factory.js

```diff
@@ -222,6 +222,27 @@ const createBeforeRequest = (app) => {
   };
 };
 
+const proxyHeaders = (headers) => {
+  const proxy = {
+    get: (target, prop) => {
+      const original = Reflect.get(target, prop);
+      if (typeof original === 'function' || typeof original === 'symbol') {
+        // defaults to defined functions on the Headers class; the symbol type
+        // is used for accessing an internal map
+        return original;
+      }
+      try {
+        // try to retrieve the header via the get() function
+        return target.get(prop); // try to retrieve the header via the get() function
```

Comment:
very very inconsequential nitpick here, but it looks like the comment above got copied twice